### PR TITLE
Add paired dataset collection support to fastq_paired_end_interlacer.

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

File: tool_collections/galaxy_sequence_utils/fastq_paired_end_interlacer/fastq_paired_end_interlacer.xml [hunk 1]
```diff
@@ -1,32 +1,64 @@
-<tool id="fastq_paired_end_interlacer" name="FASTQ interlacer" version="1.1">
+<tool id="fastq_paired_end_interlacer" name="FASTQ interlacer" version="1.2">
   <description>on paired end reads</description>
   <requirements>
     <requirement type="package" version="1.0.0">galaxy_sequence_utils</requirement>
   </requirements>
-  <command interpreter="python">fastq_paired_end_interlacer.py '$input1_file' '${input1_file.extension[len( 'fastq' ):]}' '$input2_file' '${input2_file.extension[len( 'fastq' ):]}' '$outfile_pairs' '$outfile_singles'</command>
+  <command interpreter="python">
```
Comment (gregvonkuster) [inline tool_collections/galaxy_sequence_utils/fastq_paired_end_interlacer/fastq_paired_end_interlacer.xml]: I think interpreter is deprecated here. - python $__tool_directory__/...

Comment (nsoranzo) [inline tool_collections/galaxy_sequence_utils/fastq_paired_end_interlacer/fastq_paired_end_interlacer.xml]: Done.

**Comment by bgruening** [comment]
Preparing a conda package right now, if you can wait ...

**Comment by bgruening** [comment]
@nsoranzo this package will be available in a few minutes: https://github.com/bioconda/bioconda-recipes/pull/1080

Please note that I upgrade the dep to version 1.0.1 this is an IUC repo.

**Comment by nsoranzo** [comment]
Can someone from devteam push this to the MTS? @martenson @davebx

**Comment by martenson** [comment]
@nsoranzo done, thank you !

**Pull request merged**